platform/iphone/display_server_iphone.h

```diff
@@ -41,7 +41,11 @@
 #include "vulkan_context_iphone.h"
 
 #import <QuartzCore/CAMetalLayer.h>
-#include <vulkan/vulkan_metal.h>
+#ifdef USE_VOLK
+#include <volk.h>
+#else
+#include <vulkan/vulkan.h>
```

Comment:
Is it intended that this is now `vulkan/vulkan.h` and no longer `vulkan_metal.h`? (same for `vulkan_ios.h`)

Reply:
Yes. It's auto included in the `vulkan.h` if `VK_USE_PLATFORM_IOS_MVK` is defined, which was missing for iOS and Android and required for volk to build anyway.